integration-tests/src/main/java/io/druid/testing/clients/CoordinatorResourceTestClient.java

```diff
@@ -155,6 +155,23 @@ public void deleteSegmentsDataSource(String dataSource, Interval interval)
     }
   }
 
+  public HttpResponseStatus getProxiedOverlordScalingResponseStatus()
+  {
+    try {
+      StatusResponseHolder response = makeRequest(
+          HttpMethod.GET,
+          StringUtils.format(
+              "%s/druid/indexer/v1/scaling",
+              coordinator
+          )
+      );
+      return response.getStatus();
+    }
+    catch (Exception e) {
+      throw Throwables.propagate(e);
```

Comment:
Suggest 

`throw new RE(e, "Unable to get scaling status from [%s]", coordinator)`

Reply:
Changed to the suggested message